lib/horde/registry_impl.ex

```diff
@@ -327,6 +333,18 @@ defmodule Horde.RegistryImpl do
     {:reply, :ok, state}
   end
 
+  def handle_call({:delete_meta, key, pid}, _from, state) do
+    DeltaCrdt.mutate(crdt_name(state.name), :remove, [{:registry, key}], :infinity)
+
+    for listener <- state.listeners do
+      send(listener, {:delete_meta, state.name, key, pid})
```

Comment:
This isn't something that Elixir.Registry does so we should also not do it in Horde.